torch/hub.py

```diff
@@ -136,6 +136,16 @@ def _parse_repo_info(github):
                 ref = 'master'
             else:
                 raise
+        except OSError as e:  # Accounting for timeout error such that the code loads from cache when there is no internet
```

Comment:
Should we catch a more specific `URLError` instead of `OSError`?

Reply:
Yes, that would be perfect.